backend/app/build.gradle

```diff
@@ -68,6 +68,13 @@ kotlin {
     jvmToolchain(21)
 }
 
+
+ktlint {
+    debug = true
```

Comment:
We don't need the config I guess. Do we?

Reply:
Can we keep the old version in root then?

Reply:
It is required for the new version of kotlin, otherwise fails with compile error.